bigquery/doc.go

```diff
@@ -85,6 +85,24 @@ You can also use a struct whose exported fields match the query:
 	    fmt.Println(c)
 	}
 
+The struct also can tagged using `bigquery` tag if it's not match with the field name:
```

Comment:
Maybe change to something like `Struct fields can also be tagged using the `bigquery` struct tag to match the source field name:`